packages/docs-reanimated/docs/advanced/useEvent.mdx

```diff
@@ -47,6 +105,21 @@ function useAnimatedPagerScrollHandler(handlers, dependencies) {
       }
     },
     ['onPageScroll'],
-    doDependenciesDiffer,
+    doDependenciesDiffer
   );
+}
 ```
```

Comment:
Can we please have a full example that you can copy-n-paste-n-run with imports'n'stuff 🙏 🙏 🙏